src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/contactperson/ContactPersonNomisApiService.kt

```diff
@@ -96,16 +106,18 @@ class ContactPersonNomisApiService(@Qualifier("nomisApiWebClient") private val w
     .retrieve()
     .awaitBody()
 
-  suspend fun updateContactRestriction(personId: Long, contactId: Long, contactRestrictionId: Long, request: UpdateContactPersonRestrictionRequest) = webClient.put()
-    .uri(
-      "/persons/{personId}/contact/{contactId}/restriction/{contactRestrictionId}",
-      personId,
-      contactId,
-      contactRestrictionId,
-    )
-    .bodyValue(request)
-    .retrieve()
-    .awaitBodilessEntity()
+  suspend fun updateContactRestriction(personId: Long, contactId: Long, contactRestrictionId: Long, request: UpdateContactPersonRestrictionRequest) {
```

Comment:
minor change so it returns Unit to get rid of "platform" cast warning

Reply:
I think an alternative is to just declare that it returns `Unit` i.e.
```
... : Unit = webClient.put()
```